connect/runtime/src/test/java/org/apache/kafka/connect/runtime/distributed/DistributedHerderTest.java

```diff
@@ -16,6 +16,7 @@
  */
 package org.apache.kafka.connect.runtime.distributed;
 
+import com.google.common.util.concurrent.Runnables;
```

Comment:
Are you sure this is going to work? I don't see Guava in dependencies.gradle and can't find any com.google packages anywhere in the code.

Reply:
My bad . Seems my local intellij needed some cleanup. Thanks for the catch.